code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -6656,6 +6656,24 @@
 	reagent_state = REAGENT_STATE_LIQUID
 	color = "#664300" //rgb: 102, 67, 0
 
+/datum/reagent/ethanol/deadrum/pinksnowflake
+	name = "Pink Snowflakes"
+	id = PINKSNOWFLAKE
+	description = "Do not catch pink snowflakes with your tongue. Unless you're a Plasmaman."
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#664300" //rgb: 102, 67, 0
+
+/datum/reagent/ethanol/deadrum/pinksnowflake/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+
+	var/mob/living/carbon/human/H = M
+	M.dizziness += 5
+	if(isplasmaman(H))
+		return 1
```

Comment:
I wouldn't trust this return, try turning the whole plasmaman check into a simple "if(!isplasmaman(H)) M.adjusttoxloss"

Reply:
I trust it because it works in testing (and in other places in the code, see how the "plasma" chem does it).